docs/configuration.rst

```diff
@@ -775,8 +775,12 @@ File system ID for an existing file system. Specifying this option will void all
 Config sanity will only allow file systems that: have no mount target in the stack's availability zone
 OR have existing mount target in stack's availability zone with inbound and outbound NFS traffic allowed from 0.0.0.0/0.
 
+Note: sanity check for validating efs_fs_id requires the IAM role to have permission for the following actions:
+EFS::DescribeMountTargets, EFS::DescribeMountTargetSecurityGroups, EC2::DescribeSubnets, EC2::DescribeSecurityGroups.
+Please add these permissions to your IAM role, or set sanity_check = false to avoid errors.
```

Comment:
or set sanity_check = false: here, use back-ticks. It should be rendered as `sanity_check = false`